React/Modules/RCTStatusBarManager.m

```diff
@@ -14,12 +14,25 @@
 #if !TARGET_OS_TV
 @implementation RCTConvert (UIStatusBar)
 
+#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && defined(__IPHONE_13_0) && \
```

Comment:
Is if possible to have this #ifdef just around the dark-content enum value instead of duplicating the whole thing?

Reply:
Fixed